src/NLog/GlobalDiagnosticsContext.cs

```diff
@@ -33,27 +33,25 @@
 
 namespace NLog
 {
+    using NLog.Context;
+    using NLog.Helpers;
     using System;
     using System.Collections.Generic;
 
     /// <summary>
     /// Global Diagnostics Context - a dictionary structure to hold per-application-instance values.
     /// </summary>
+    [Obsolete("Use NLog.Context.GlobalContext Instead.")]
```

Comment:
Nice, the changes in `GlobalDiagnosticsContext` are just as I expected (obsolete attr, invoking new class)